packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -335,13 +344,16 @@ class IconButton extends StatelessWidget {
     else
       currentColor = disabledColor ?? theme.disabledColor;
 
-    final VisualDensity effectiveVisualDensity = visualDensity ?? theme.visualDensity;
+    final VisualDensity effectiveVisualDensity =
+        visualDensity ?? theme.visualDensity;
```

Comment:
please avoid this kind of spurious change, it's more readable on one line

Reply:
Sorry, this was auto format as well